bin/windows/mssql_db_create.sql

```diff
@@ -0,0 +1 @@
+IF db_id('soci_test') IS NULL BEGIN CREATE DATABASE [soci_test] END;
```

Comment:
Is the test for the database existence really necessary? AFAIU we start from a clean slate every time with AppVeyor, so it shouldn't already exist -- and if it somehow does, it could presumably contain something that could interfere with our tests, so perhaps it would be better to delete and recreate it?